cornflow/endpoints/user.py

```diff
@@ -174,3 +173,53 @@ def put(self, user_id, make_admin):
             db.session.commit()
 
         return user_obj, 200
+
+
+class RecoverPassword(MetaResource, MethodResource):
+    @doc(description="Send email to create new password", tags=["Users"])
+    @use_kwargs(RecoverPasswordRequest)
+    def put(self, **kwargs):
+        """
+        API method to send an email to the user if they forgot to password.
+        Sends a temporary password and updates the database.
+        :param kwargs: a dictionary containing the email address
+        :return: A dictionary with a message (error if the email address is not in the database) and an integer with
+            the HTTP status code.
+        :rtype: Tuple(dict, integer)
+        """
+        email = kwargs.get("email")
+
+        email_config = {
+            "email_sender": current_app.config["CORNFLOW_EMAIL_ADDRESS"],
+            "password": current_app.config["CORNFLOW_EMAIL_PASSWORD"],
+            "server": current_app.config["CORNFLOW_EMAIL_SERVER"],
+            "port": current_app.config["CORNFLOW_EMAIL_PORT"],
+            "email_receiver": email,
+        }
+
+        if (
+            email_config["email_sender"] is None
+            or email_config["password"] is None
+            or email_config["server"] is None
+            or email_config["port"] is None
+            or email_config["email_receiver"] is None
+        ):
+            raise InvalidUsage(
+                "This functionality is not available. Check that cornflow's email is correctly configured"
+            )
+
+        message = f"A new password was generated for user with email: {email}"
+
+        if not UserModel.check_email_in_use(email):
+            return {"message": message}, 200
```

Comment:
I don't think this is right: the condition checks if the email corresponds to an user in the database. If it doesn't, it should return an error, or at least a warning saying that the password could not be changed because the user wasn't found

Reply:
The thing is I do not want to give information about which emails exist on the database and which don't. That is why the message back is the same. I think I'm going to modify it to make it more standard, but I do not want to make that distinction

Reply:
Okay, that makes sense